### PR TITLE
iOS-3826 Xcode | fix warnings

### PR DESCRIPTION
**Pull request by mgolovko**

### CODE REVIEW

File: Anytype/Sources/Analytics/AnytypeAnalytics/AnytypeAnalytics.swift [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import Foundation
 import Services
-
+import AnytypeCore
```
Comment (ignatovv) [inline Anytype/Sources/Analytics/AnytypeAnalytics/AnytypeAnalytics.swift:3]: unnecessary import?

File: Anytype/Sources/PresentationLayer/Common/SwiftUI/StatusBarStyle/StatusBarStyleModifier.swift [hunk 2]
```diff
@@ -26,6 +26,7 @@ struct StatusBarStyleModifier: ViewModifier {
     
     let style: AnytypeStatusBarStyle
     
+    @available(*, deprecated) // Workround for disable deprecated API
```
Comment (ignatovv) [inline Anytype/Sources/PresentationLayer/Common/SwiftUI/StatusBarStyle/StatusBarStyleModifier.swift:29]: Workround for disable deprecated API **warning**

**Review by ignatovv — APPROVED** [review]

File: Anytype/Sources/PresentationLayer/Common/SwiftUI/StatusBarStyle/StatusBarStyleModifier.swift (hunk 2 above)
Comment (mgolovko) [reply Anytype/Sources/PresentationLayer/Common/SwiftUI/StatusBarStyle/StatusBarStyleModifier.swift]: fixed

File: Anytype/Sources/Analytics/AnytypeAnalytics/AnytypeAnalytics.swift (hunk 1 above)
Comment (mgolovko) [reply Anytype/Sources/Analytics/AnytypeAnalytics/AnytypeAnalytics.swift]: fixed

**Pull request merged**